docs/dev-env-setup.md

```diff
@@ -206,52 +206,23 @@ found 6 vulnerabilities (2 low, 4 high)
   run `npm audit fix` to fix them, or `npm audit` for details
 ```
 
-Then we will compile the assets with `grunt`.
+Then we will compile the assets with `webpack` by running `npm run prod`.
```

Comment:
here we need to run `npm run build`, it'll run `webpack --config ./webpack/webpack.dev.js`
(`npm run prod` compiles the production build with minification and some optimizaitions for production)

Reply:
Actually I've just noticed this part exists already:
https://github.com/webcompat/webcompat.com/blob/456791f3d6e4076f6f3f304349bbb887a3a29913/docs/dev-env-setup.md#installing-npm-dependencies
Maybe we can delete it then?

Reply:
removed the assets section